realm/kotlin-extensions/build.gradle

```diff
@@ -196,7 +196,9 @@ publishing {
             groupId 'io.realm'
             artifactId 'realm-android-kotlin-extensions'
             version project.version
-            artifact file("${rootDir}/kotlin-extensions/build/outputs/aar/realm-kotlin-extensions-base-release.aar")
+            artifact (file("${rootDir}/kotlin-extensions/build/outputs/aar/realm-kotlin-extensions-base-release.aar")) {
+                builtBy assemble
```

Comment:
In order to keep the build times low, I think doing `assembleRelease` here should be enough. If we just do `assemble` it will force a built of both debug and release variants on CI which can be quite time consuming.

Reply:
It doesn't work when I specify assembleRelease:
```
Could not get unknown property 'assembleRelease' for FileBasedMavenArtifact_Decorated aar:null of type org.gradle.api.publish.maven.internal.artifact.FileBasedMavenArtifact.
```

Reply:
I guess this happens because the buildtype can't be used to define the precedence. If you chose to build in debug mode, then it would use `assembleDebug`, if it's release then it would choose `assembleRelease`.

Reply:
Cool, should be good enough 👍

Reply:
My previous comment isn't actually true. I just checked the log from the build process and both before these changes and after, the task in question is `assemble`, no build type specified.

Also, the build times don't seem to be affected by this. Before, the time for the "build" stage of the job oscillated between 6-10 minutes, and now it's exactly the same.